lib/DDG/Spice/SkiResorts.pm

```diff
@@ -0,0 +1,34 @@
+package DDG::Spice::SkiResorts;
+
+# ABSTRACT: Displays information on ski resorts
+
+# Start at http://docs.duckduckhack.com/walkthroughs/forum-lookup.html if you are new
+# to instant answer development
+
+use DDG::Spice;
+use JSON;
+
+# Configuration
+spice is_cached => 1;
+spice proxy_cache_valid => "200 1d"; # defaults to this automatically
+spice wrap_jsonp_callback => 1; # only enable for non-JSONP APIs (i.e. no &callback= parameter)
+
+# API endpoint
+spice to => 'http://www.piste.io/info/$1.json';
+
+# Load in resort list for triggers
+my $data = share('resorts.json')->slurp;
+my @resorts = @{decode_json($data)};
+@resorts = map { $_ =~s/-/ /g; $_; } @resorts; # Replace dashes with spaces
+triggers any => @resorts;
```

Comment:
I'm worried that this is going to be way too broad. This would trigger on any query that contains the name of any resort. It's very likely that not every query in the set would be well served by this Instant Answer.

Let's use trigger words that better suggest user intent. E.g a `start` trigger for "ski conditions at", "ski conditions for", and an `end` trigger for `ski conditions`.

Alternatively, we could use a regular expression in the `handle` that verifies the query follows one of those query patterns.